modules/ROOT/pages/connected-apps-developers.adoc

```diff
@@ -0,0 +1,86 @@
+= Connected Apps for Developers
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Platform provides a https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[comprehensive set of APIs] that that enables developers to extend its functionality.
+With the Connected Apps feature, developers can delegate access to applications that interact with Anypoint Platform programmatically using APIs.
+While some might use the Connected Apps feature to build CI/CD pipelines, others can productize additional third-party use-cases on top of Anypoint Platform.
+
+In addition to enabling new flows to authenticate users, developers can enable or disable clients and view client usage.
+For example, developers can see how many organizations are using the app and how many users have access.
+
+The Connected Apps feature supports OAuth 2.0 and OpenID Connect. As such, developers must ensure their app can request access to user data using tokens.
```

Comment:
This whole paragraph reads weird to me. Can we remove it entirely? I'm not sure what information it's trying to convey other than "your app has to be an oauth app"